.vscode/settings.json

```diff
@@ -1,4 +0,0 @@
-{
-    "python.analysis.stubPath": "/Applications/Talon.app/Contents/Resources/python/lib/python3.9/site-packages",
-    "python.formatting.provider": "black"
```

Comment:
I wonder if there is any way to preserve this line?  I believe the default path for `python.analysis.stubPath` is `typings` subdir.  What if we did the following:

- Remove line 2 here, but otherwise leave this file
- add `typings` to `.gitignore`

Then I can just symlink `typings` to `/Applications/Talon.app/Contents/Resources/python/lib/python3.9/site-packages` locally.  WDYT?

Reply:
I don't have an operating system that supports symlinks. I can't have the file there because I will get conflicts

Reply:
Right I see so you'd like to use your own `.vscode/settings.json` to set a `python.analysis.stubPath` that works for your OS?

Reply:
Correct